Code/RDGeneral/RDValue-taggedunion.h

```diff
@@ -44,6 +44,7 @@
 #include <any>
 #include <boost/utility.hpp>
 #include <boost/lexical_cast.hpp>
+#include <boost/numeric/conversion/cast.hpp>
```

Comment:
The docs seem to recommend: 
```suggestion
#include <boost/numeric_cast.hpp>
```
Is there a reason you chose the more specific include?

Reply:
I went with the same header already referenced in `rdkit`: 
https://github.com/rdkit/rdkit/blob/a0f274c0f7575d8b7e0ac0585152f4aa924bf26a/Code/RDGeneral/Invariant.h#L29